go/tools/wtool/main.go

```diff
@@ -0,0 +1,144 @@
+/*wtool augments your bazel WORKSPACE file with new_go_repository entries
+
+Example Usage: wtool com_github_golang_glog com_google_cloud_go
```

Comment:
If you put the example command on a new line and indent it, godoc will put it in a box when rendered to HTML.

e.g. https://godoc.org/golang.org/x/tools/cmd/godoc https://github.com/golang/tools/blob/master/cmd/godoc/main.go